cosmic_ray/operators/remove_decorator.py

```diff
@@ -0,0 +1,27 @@
+import ast
+
+from .operator import Operator
+
+
+class RemoveDecorator(Operator):
+    """An operator that removes each of the non standard decorators."""
+    regular_decorators = ["classmethod", "staticmethod", "abstractmethod"]
+
+    def visit_FunctionDef(self, node):  # noqa
+        decorator_candidates = [x for x in node.decorator_list if \
```

Comment:
These lines breaks are not necessary. And I think it would read better if you put the `if` on the same line as the rest of the conditional handling:
```
decorator_candidates = [x for x in node.decorator list
                                          if x.id not in self.regular_decorators]
```